src/BenchmarkDotNet/Columns/FirstCallColumn.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
```

Comment:
The `System`, `System.Collections.Generic`, and `System.Text` namespaces are unused, they can be safily removed.